WORKING_GROUPS.md

```diff
@@ -248,3 +249,18 @@ Responsibilities include:
   to be generated when needed.
 * Defining and adding common structures to the dumps generated
   in order to support tools that want to introspect those dumps.
+
+### [Release](https://github.com/nodejs/release)
+
+The purpose of the release Working Group is to manage/execute
```

Comment:
Nit: Remove the passive voice here. Also, `release Working Group` -> `Release Working Group`. Also, `release...process for all releases` is a head-scratcher. And this sentence needs a period at the end.

Here's what I'd do:

> The Release Working Group manages the release process for Node.js. Responsibilities include: